public/app/percona/backup/components/BackupInventory/BackupInventory.service.ts

```diff
@@ -32,6 +32,13 @@ export const BackupInventoryService = {
       })
     );
   },
+  async restore(serviceId: string, locationId: string, artifactId: string): Promise<any> {
```

Comment:
Please, add the type of the return value

Reply:
👍